akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -1,5 +1,42 @@
 # Migration Guide 2.5.x to 2.6.x
 
+## Default remoting is now Artery TCP
+
+@ref[Artery TCP](../remoting-artery.md) is now the default remoting implementation. 
+Classic remoting has been deprecated and will be removed in `2.7.x`.
+To migrate to Artery a full cluster restart is required. If you've already moved to Artery in 2.5.x
+then no changes are required and a normal rolling restart is possible.
+
+Configuration for artery is under `akka.remote.artery` configuration for classic remoting in
+`2.6` has moved from `akka.remote` to `akka.remote.classic`. Configuration that is used for both
+remains under `akka.remote`. 
+
+### Switching to Artery
+
+To switch to Artery any overrides for classic remoting need to be ported to Artery configuration.
+The most likely are:
+
+Hostname and port for binding:
+
+* `akka.remote.netty.tcp.hostname` => `akka.remote.artery.canonical.hostname`
+* `akka.remote.netty.tcp.port`=> `akka.remote.artery.canonical.port`
+
```

Comment:
You can move/incorporate the "Migrating from classic remoting" section in remoting-artery.md to here